ext/snmp/snmp.c

```diff
@@ -87,6 +87,13 @@ typedef struct snmp_session php_snmp_session;
 	} \
 }
 
+#define PHP_SNMP_FETCH_OBJECT 								\
```

Comment:
I would prefer a macro function rather than assume the name of the variable

Reply:
I'm also not a big fan of these obscure macros, and given that we need it only once for now, I would suggest to inline it in `::setSecurity()`.

Reply:
in fact I was waiting feedback on the close() comment.